recipes/jerryscript/all/conanfile.py

```diff
@@ -41,7 +88,23 @@ def configure(self):
         del self.settings.compiler.cppstd
         if self.options.shared:
             del self.options.fPIC
-            
+
+        if not self.options.debugger:
+            del self.options.keep_line_info
+        predefined_profiles = ["es.next", "es5.1", "minimal"]
+        if (not self.options.profile in predefined_profiles) and not os.path.isfile(self.options.profile):
```

Comment:
I'm not entirely happy about accepting an absolute path as an option.
This might lead to reproducible builds.

I would limit the options to `"profile": ["es.next", "es5.1", "minimal", "conan-environment"]`,
Then, if `self.options.profile == "conan-environment"`, fetch the profile from the environment variable named `JERRYSCRIPT_CONAN_PROFILE_PATH`.

This might also lead to reproducible builds when setting the environment variable from the command prompt,
but it has the advantage that another recipe in the conan profile can set this environment variable.

Reply:
I can give a try at overriding package id, to use the file contents for the package option. That would realistically reflect compatibility. Should I still make the change regarding the environment variable?

Reply:
Nice idea.
But I think it's okay to leave it as it is now.
Perhaps another purist will fix it.